### PR TITLE
[CI] Upgrade generic-worker version used on mac workers

### PR DESCRIPTION
**Pull request by djmitche**
(As requested in #231)

### CODE REVIEW

**Comment by glandium** [comment]
The worker fails to start with
```
$ CI/start-worker.sh worker-$TC_WORKER_TYPE $TC_WORKER_TYPE

Missing key

The command "CI/start-worker.sh worker-$TC_WORKER_TYPE $TC_WORKER_TYPE" exited with 1.
```

**Comment by djmitche** [comment]
I suspect that's because it's a PR and doesn't have access to the Travis-CI secrets.

**Comment by glandium** [comment]
Oh right... Let me push this on the try branch.

**Comment by glandium** [comment]
```
$ CI/start-worker.sh worker-$TC_WORKER_TYPE $TC_WORKER_TYPE

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current

                                 Dload  Upload   Total   Spent    Left  Speed

100   617    0   617    0     0   2452      0 --:--:-- --:--:-- --:--:--  2458

100 13.4M  100 13.4M    0     0  8871k      0  0:00:01  0:00:01 --:--:-- 17.1M

2019/10/28 23:50:14 Loading generic-worker config file '/Users/travis/build/glandium/git-cinnabar/worker.config'...

2019/10/28 23:50:14 Creating file /Users/travis/build/glandium/git-cinnabar/worker.config...

2019/10/28 23:50:14 Saving file /Users/travis/build/glandium/git-cinnabar/worker.config (absolute path: /Users/travis/build/glandium/git-cinnabar/worker.config)

2019/10/28 23:50:14 Invalid config: Config setting "rootURL" has not been defined

2019/10/28 23:50:14 Exiting worker with exit code 73

The command "CI/start-worker.sh worker-$TC_WORKER_TYPE $TC_WORKER_TYPE" exited with 73.
```

**Comment by djmitche** [comment]
Ah, I missed copying over the rootURL change.

**Comment by glandium** [comment]
```
(Permanent) HTTP response code 422

HTTP/1.1 422 Unprocessable Entity
```
https://api.travis-ci.org/v3/job/604229551/log.txt

**Comment by glandium** [comment]
```
Method: POST

URL: https://github.com/mozilla/community-tc-config/api/queue/v1/claim-work/proj-git-cinnabar/osx-10-10
```
This doesn't seem right.

**Comment by djmitche** [comment]
Wow, that was a boneheaded copy/paste error.  Firefox fails to copy in X like 10% of the time, and sometimes I forget to check.

File: CI/start-worker.sh [hunk 1]
```diff
@@ -27,13 +27,14 @@ cat > worker.config <<EOF
   "requiredDiskSpaceMegabytes": 512,
   "signingKeyLocation": "worker.key",
   "tasksDir": "tasks",
+  "rootURL": "https://community-tc.services.mozilla.com",
```
Comment (glandium) [inline CI/start-worker.sh]: mmmm that's not going to work without #231, is it?

Comment (glandium) [reply CI/start-worker.sh]: Is there a url that would work with the current tc instance?

Comment (djmitche) [reply CI/start-worker.sh]: Ah, `https://taskcluster.net` should do.

Sorry for the silly mistakes -- it's a bit past my bedtime :)

**Comment by djmitche** [comment]
That's looking more promising..

**Pull request merged**